packages/katello/katello/katello.cron

```diff
@@ -0,0 +1,14 @@
+SHELL=/bin/sh
+PATH=/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin
+
+# Script for initiating removal of orphaned content
+00 22 * * 0	root	foreman-rake katello:delete_orphaned_content RAILS_ENV=production >/dev/null 2>&1
+
+# cronjob to clean puppet envs for puppet 4
+15 22 * * 0	root	[ -d /etc/puppetlabs/code/environments ] &&  find /etc/puppetlabs/code/environments/KT* -maxdepth 0 -type d -empty -delete > /dev/null 2>&1
+
+# cronjob to clean puppet envs for puppet 3
+20 22 * * 0	root	[ -d /etc/puppet/environments ] && find /etc/puppet/environments/KT* -maxdepth 0 -type d -empty -delete > /dev/null 2>&1
+
+# Script for publishing unpublished repositories
+* 22 * * *     root    foreman-rake katello:publish_unpublished_repositories RAILS_ENV=production >/dev/null 2>&1
```

Comment:
Doesn't this mean every minute of the 22nd hour?

Reply:
yes, it does [see here](http://corntab.com/?c=*_22_*_*_*_)

looks like we need to update to `0 22 * * * ` [see here](http://corntab.com/?c=0_22_*_*_*_)